imports/plugins/included/payments-stripe/server/methods/stripeapi-methods-capture.app-test.js

```diff
@@ -149,13 +149,9 @@ describe("stripe/payment/capture", function () {
       return "sk_fake_fake";
     });
 
-    let captureResult = null;
-    let captureError = null;
     Meteor.call("stripe/payment/capture", paymentMethod, function (error, result) {
-      captureResult = result;
-      captureError = error;
-      expect(captureError.message).to.equal("Match error: Match error: Transaction id is required");
-      expect(captureResult).to.be.undefined;
+      expect(error.message).to.equal("[Transaction ID is required]");
```

Comment:
They are no longer "Match error"s since they don't go through the `check` package